src/ducks/balance/helpers.js

```diff
@@ -148,5 +148,5 @@ export const getGroupBalance = group => {
     return 0
   }
 
-  return sumBy(group.accounts.data, account => get(account, 'balance') || 0)
+  return sumBy(group.accounts.data, account => get(account, 'balance', 0))
```

Comment:
you can use accounts:
```
    return sumBy(accounts, account => get(account, 'balance', 0))
```

Reply:
![](https://media.giphy.com/media/4WKypHPsEoppe/giphy.gif)